pyvista/plotting/plotting.py

```diff
@@ -811,7 +811,7 @@ def key_press_event(self, obj, event):
                 for func in callbacks:
                     func()
         except Exception as e:
-            log.error('Exception encountered for keypress "%s": %s' % (key, e))
+            log.error('Exception encountered for keypress "%s": %s', key, e)
```

Comment:
Is this right? Are the arguments of `log.error` after the message things to format? Should we just use the `.format()` method?

Reply:
Any args for a log string are actually recommended to be input as args.  Otherwise, if there's an error with string formatting (like `key` is missing as it sometimes happens on my end), then the logger will report the error as a warning instead of an exception (which is disastrous in a `Qt` app.  We could implement another try/catch or see if key exists in locals, but I figured to kill two birds with one stone.